src/ape/api/networks.py

```diff
@@ -246,10 +246,13 @@ def use_provider(
             provider_name = provider_name.split(":")[0]
 
         if provider_name in self.providers:
-            return ProviderContextManager(
-                self.ecosystem.network_manager,
-                self.providers[provider_name](provider_settings=provider_settings),
-            )
+            try:
+                return ProviderContextManager(
+                    self.ecosystem.network_manager,
+                    self.providers[provider_name](provider_settings=provider_settings),
+                )
+            except TypeError as e:
+                raise Exception(f"Plugin {provider_name} failed to load!")
```

Comment:
I left it as `Exception` for consistency with the rest of the file, should I refactor it to `Abort`?

Reply:
`e` is not used, I could include it something like: 
```            
except TypeError as e:
    raise Exception(f"Plugin {provider_name} failed to load!",e)```

Reply:
this also doesn't note the failure and skip loading that plugin module.

Reply:
Let's talk more about how to fix this one